### ISSUE TITLE
Massive amount of issues on official specification W3C WAI-ARIA pages

### ISSUE THREAD

**Issue by DiegoBM-DB** [dequelabs/axe-core#1272]
Does it make sense that the W3C pages for accessibility score more than 4000 errors in the axe report?
Page being tested: https://www.w3.org/TR/wai-aria-practices

Expectation: 0 issues

Actual: 4612 issues

Motivation: Technically axe is supposed to be following WAI guidelines. Is W3C following them as well or is this a bug in extension?

axe-core version: 3.1.1
axe chrome extension version: 3.6.0

**Comment by marcysutton** [comment]
If you look at the results, the majority of them are review items for color contrast–not errors or violations. If you turn the decorative background-image off from the body element and run again, they mostly go away except for a handful of legit color contrast violations like on the "Table of Contents" header which has a ratio of 2.9:1. There are still a few contrast review items and some legit issues with links not having accessible names until you focus them. There is also a missing `main` landmark, which is a best practice. 

Axe is behaving normally, so I'm going to close the issue. But thank you for opening it!

**Comment by DiegoBM-DB** [comment]
Thank you for clarifying it @marcysutton
